snap-core/src/main/java/org/esa/snap/core/util/ImageUtils.java

```diff
@@ -249,19 +249,26 @@ public static Dimension computeSceneRasterSize(int defaultSceneRasterWidth, int
     }
 
     public static int computeTileCount(int imageSize, int tileSize) {
-        int tileCount = imageSize / tileSize;
+        /*int tileCount = imageSize / tileSize;
```

Comment:
Please remove old code and do not comment it out.

Here a unit level test would be good,